src/components/tables/DataTable/DataTableStream.stories.tsx

```diff
@@ -0,0 +1,247 @@
+              }
+            />
+          }
+          {...dataTableProps}
+        />
+      </DataTableStream.TableProviderStream>
+    </Panel>
+  );
+};
+
+// Column definitions
+const columnDefinitions = [
+  {
+    id: 'name',
+    accessor: (user: User) => user.name,
+    Header: 'Name',
+    Cell: ({ value }: { value: string }) => value,
+    disableSortBy: false,
+    disableGlobalFilter: false,
+  },
+  {
+    id: 'email',
+    accessor: (user: User) => user.email,
+    Header: 'Email',
+    disableSortBy: false,
+    disableGlobalFilter: false,
+  },
+  {
+    id: 'company',
+    accessor: (user: User) => user.company,
+    Header: 'Company',
+    disableSortBy: false,
+    disableGlobalFilter: false,
+  },
+  {
+    id: 'joinDate',
+    accessor: (user: User) => user.joinDate,
+    Header: 'Joined',
+    Cell: ({ value }: { value: Date }) =>
+      value.toLocaleDateString('en-US', { year: 'numeric', month: 'long', day: 'numeric' }),
+    sortType: sortDateTime,
+    disableSortBy: false,
+    disableGlobalFilter: true,
+  },
+];
+
+// Stories
+export const Empty = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 0 }),
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const SinglePage = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 10 }),
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const MultiplePagesSmall = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 45 }),
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const MultiplePagesLarge = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 1000 }),
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const SlowNetwork = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 1000 }),
+    delay: 1500,
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const InfiniteDelay = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 50 }),
+    delay: Number.POSITIVE_INFINITY,
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const StatusFailure = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 1000 }),
+    delay: -1,
+  },
+  render: (args: dataTeableLazyTemplateProps) => <DataTableStreamTemplate {...args} />,
+};
+
+export const WithEndOfTablePlaceholder = {
+  args: {
+    columns: columnDefinitions,
+    items: generateData({ numItems: 15 }),
+    dataTableProps: {
+      placeholderEndOfTable: <Banner variant="info">You have reached the end of the table</Banner>
```

Comment:
![Screenshot 2025-01-15 at 12 22 33](https://github.com/user-attachments/assets/7d54e918-db04-48af-ae8b-cb16c1af35fe)

This is rendering with the default "Info" title. Looks a bit weird in this context though. Maybe we should just move the message to the title here and leave the content blank?

![Screenshot 2025-01-15 at 12 23 33](https://github.com/user-attachments/assets/14664a52-0280-4741-8d0d-307c546ac58a)